docs/en_US/rulesets.md

```diff
@@ -3,26 +3,26 @@
 This page describes how to write rulesets for [HTTPS
 Everywhere](https://eff.org/https-everywhere), a browser extension that
 switches sites over from HTTP to HTTPS automatically. HTTPS Everywhere comes
-with [thousands](http://www.eff.org/https-everywhere/atlas/) of rulesets that
+with [thousands](https://www.eff.org/https-everywhere/atlas/) of rulesets that
```

Comment:
Shall we use the new atlas URL https://atlas.eff.org/index.html as per https://github.com/EFForg/https-everywhere/pull/17674#issue-267149300 here?